File: TouristAttractionsSpringThrift/Spring/TouristAttractions/AttractionsModel/src/main/java/model/AgencyUser.java
package model;

import java.util.Objects;

public class AgencyUser extends Entity<Integer>{
    private String userName;
    private String password;

    public AgencyUser(int id, String userName, String password) {
        this.userName = userName;
        this.password = password;
        this.setId(id);
    }

    public String getUserName() {
        return userName;
    }

    public String getPassword() {
        return password;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        AgencyUser that = (AgencyUser) o;
        return Objects.equals(getId(), that.getId());
    }

    @Override
    public String toString() {
        return "AgencyUser{" +
                "name='" + userName + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}

File: TouristAttractionsSpringThrift/Thrift/CSharp/TouristAttractionsCS/Model/model/AgencyUser.cs
using System;

namespace TouristAttractions.model
{
    [Serializable]
    public class AgencyUser : Entity<int>
    {
        public AgencyUser()
        {
        }

        public AgencyUser(int Id, String UserName, String Password)
        {
            this.UserName = UserName;
            this.Password = Password;
            this.Id = Id;
        }

        public string UserName { get; set;}
        public string Password{ get; set;}

        public override string ToString()
        {
            return "UserName: " + UserName + " Password: " + Password;
        }
    }
}
